store/src/java-test/com/zimbra/cs/filter/NotifyTest.java

```diff
@@ -112,4 +112,37 @@ public void filterValidToField() {
 
     }
 
+    @Test
+    public void testNotifyMailtoWithHeaderVariable() {
+        String sampleMsg = "from: abc@zimbra.com\n"
+                + "Subject: Hello\n"
+                + "to: test@zimbra.com\n";
+        String filterScript = "require [\"enotify\", \"variables\"];\n"
+            + "set \"to\" \"nick\";\n"
+            + "if anyof (header :contains [\"Subject\"] \"Hello\") {\n"
+            + "notify \"test2@zimbra.com\" \"\" \"${SUBJECT} ${to}\"\n"
```

Comment:
Existing automation has been executed for notify action (legacy version). Also, the unit test has been added for legacy notify action.